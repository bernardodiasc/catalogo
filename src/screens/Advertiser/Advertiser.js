import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import config from 'config'

import Layout from 'displays/Layout'

class Advertiser extends Component {
  static defaultProps = {
    city: null,
    advertiser: null
  }

  render() {
    return (
      <Layout sidebar={false}>
        {this.props.advertiser}
        <hr />
        <Link to={`${config.PUBLIC_URL}/${this.props.city}`}>voltar para pagina da cidade</Link><br/>
      </Layout>
    )
  }
}

export default Advertiser
