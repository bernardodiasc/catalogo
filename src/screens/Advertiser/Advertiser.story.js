import React from 'react'
import { storiesOf } from '@storybook/react'
import { withKnobs } from '@storybook/addon-knobs/react'
import StoryRouter from 'storybook-router'
import Provider from 'stories/containers/Provider'
import Docs from 'stories/containers/Docs'

import Advertiser from './Advertiser'
import AdvertiserReadme from './README.md'
storiesOf('Screens/Advertiser', module)
  .addDecorator(story => <Provider story={story()} />)
  .addDecorator(StoryRouter())
  .addDecorator(withKnobs)
  .add('Info', () => <Docs>{AdvertiserReadme}</Docs>)
  .add('Example', () => <Advertiser />)
  .add('Minimal render', () => <Advertiser />)
