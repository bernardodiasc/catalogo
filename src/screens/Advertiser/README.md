# Advertiser

## How to use

```
import Advertiser from 'screens/Advertiser'
```

```
<Advertiser />
```

## Properties

| propName | propType | defaultValue | isRequired |
| - | - | - | - |
