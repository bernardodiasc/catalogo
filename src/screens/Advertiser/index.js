import Advertiser from './Advertiser'
export default Advertiser
