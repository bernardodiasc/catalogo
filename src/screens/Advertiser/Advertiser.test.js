import React from 'react'
import { shallow } from 'enzyme'

import Advertiser from './Advertiser.js'

it('renders without crashing', () => {
  const component = shallow(<Advertiser />)
  expect(component).toMatchSnapshot()
})
