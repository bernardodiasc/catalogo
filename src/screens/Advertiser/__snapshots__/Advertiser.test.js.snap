// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Layout
  headbar={true}
  notification={false}
  sidebar={false}
  toggleHeadbar={[Function]}
  toggleSidebar={[Function]}
>
  <hr />
  <Link
    replace={false}
    to="/null"
  >
    voltar para pagina da cidade
  </Link>
  <br />
</Layout>
`;
